dev/up

```diff
@@ -8,7 +8,7 @@ if ! which golangci-lint &>/dev/null; then brew install golangci-lint; fi
 if ! which shellcheck &>/dev/null; then brew install shellcheck; fi
 if ! which protoc &>/dev/null; then brew install protobuf; fi
 if ! which protoc-gen-go &>/dev/null; then go install google.golang.org/protobuf/cmd/protoc-gen-go@latest; fi
-if ! which mockgen &>/dev/null; then go install go.uber.org/mock/mockgen@latest; fi
```

Comment:
This one was a pain to find, but tests were failing until I did this.

Basically, it's entirely possible you have a different version of `mockgen` installed on your system (in fact, you're guaranteed to have it if you have developed against `xmtp-node-go` in the past). That would be github.com/golang/mock/mockgen instead of this uber one.

The reason for the use of the uber one seems to be the `.Satisfied` property, which lets you check on an interval if all of the expected API calls were performed on the mock. We're using it in tests like this:

```
require.Eventually(t, ctrl.Satisfied, 5*time.Second, 100*time.Millisecond)
```

Reply:
Nice catch